project/Hooje/README.md

```diff
@@ -13,29 +13,29 @@ For each node, use the same method to decide the rule and create its child nodes
 # Problem to Solve
 Decision Tree is an effective model in many classification senerio. However, it takes so much time to run because it cannot be speedup by CUDA like Neural Network, especially when there are a lot of features in that Dataset.
 
-Thus, I want to implement a simple version of Decision Tree in C++ (can be used from Python) from scratch to reduce the execution time. Besides, I would like to modify some of its functions so that it can be executed parallelly.
+
+Thus, I want to implement a simple version of Decision Tree in C++ (can be used from Python) from scratch to reduce the execution time. 
+Besides, I will compare the results between with and without SIMD.
+After that, I will try to finish multi-thread version and compare it with them.
 
 # Prospective Users
 Those people who use traditional ML models can benefit from my Decision Tree. 
 They can spent less time waiting for the model to finish.
 
 # System Architecture
-My Decision Tree will be a class in C++.
+There will be a Dataset class, which will include features and its value.
+And a Node class, which represents each node in the Tree and connects with its parent node and child node. 
+And a Decision Tree Class, which will be the whole structure of Decision Tree and provide functions of Deicsion Tree.

```

Comment:
Do you plan to draw a class diagram to describe their relationship?

Reply:
I haven't thought about using diagram to visualize their relationship.
It sounds like a good idea,
I'll work on it, thanks.